### ISSUE TITLE
Since updating, font size is inconsistent

### ISSUE THREAD

**Issue by kira-m** [microsoft/vscode#101957]
- VSCode Version: 1.46.1
- OS Version: mac OS Catalina, 10.15.3

Steps to Reproduce:

Since updating, the font size has been inconsistent between files. Files with fewer lines of code render with very large font sizes. If I try to zoom out, the font size is so small it's unreadable in bigger files. I have inspected with dev tools, and even with `editor.fontSize: 13`, the HTML in the picture below is at 28px.

I've tried:

- Reinstalling
- Disabling all extensions
- Removing the font family setting, and removing the font ligatures

Here's my settings: 
```
{
    "workbench.colorTheme": "Horizon Italic",
    "editor.wordWrap": "on",
    "workbench.startupEditor": "none",
    "explorer.confirmDelete": false,
    //"editor.fontLigatures": true,
    //"editor.fontFamily": "FiraCode-Retina",
    "explorer.confirmDragAndDrop": false,"editor.detectIndentation": true,
    "editor.tabSize": 2,
    "terminal.external.osxExec": "iTerm.app",
    "terminal.explorerKind": "external",
    "terminal.integrated.shell.osx": "/bin/zsh",
    "workbench.editor.enablePreview": false,
    "workbench.editor.enablePreviewFromQuickOpen": false,
    "breadcrumbs.enabled": true,
    "explorer.openEditors.visible": 0,
    "window.title": "${dirty} ${activeEditorMedium}${separator}${rootName}",
    "editor.minimap.renderCharacters": false,
    "editor.minimap.maxColumn": 200,
    "editor.minimap.showSlider": "always",
    "editor.smoothScrolling": true,
    "editor.cursorBlinking": "phase",
    "editor.cursorSmoothCaretAnimation": true,
    "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true,
    "workbench.editor.highlightModifiedTabs": true,
    "javascript.updateImportsOnFileMove.enabled": "always",
    "diffEditor.renderSideBySide": false,
    "files.exclude": {
        "**/.classpath": true,
        "**/.project": true,
        "**/.settings": true,
        "**/.factorypath": true,
        "**/.vscode": true,
    },
    "todo-tree.highlights.enabled": true,
    "eslint.run": "onSave",
    "typescript.updateImportsOnFileMove.enabled": "always",
    "task.quickOpen.history": 5,
    "workbench.sideBar.location": "right",
    "editor.formatOnPaste": true,
    "files.trimFinalNewlines": true,
    "editor.minimap.enabled": false,
    "explorer.compactFolders": false,
    "editor.renderWhitespace": "none",
    "window.zoomLevel": 0,
    "editor.fontSize": 13,
    "tabout.disableByDefault": false,



}
```

Does this issue occur when all extensions are disabled?: Yes.

<img width="1680" alt="Screen Shot 2020-07-08 at 4 38 19 PM" src="https://user-images.githubusercontent.com/6858374/86977441-d9d2f800-c139-11ea-84c4-023589c53af5.png">

**Comment by alexdima** [comment]
@kira-m 

Can you please check both your user settings and workspace settings? It looks like some language specific setting defines the font size to be larger for HTML files.

Does this reproduce when disabling all extensions?  Press `F1` and then choose `Developer: Reload Window with Extensions Disabled`

![Kapture 2020-07-20 at 15 28 23](https://user-images.githubusercontent.com/5047891/87943040-c3b02a80-ca9d-11ea-847d-5f850fae8f05.gif)

**Comment by kira-m** [comment]
@alexdima 

Unfortunately it's not only HTML files. It seems to be based on how many lines are in the file - the font is bigger on shorter files. 

It does persist with extensions disabled; I've attached a screenshot which was taken with extensions disabled. The fonton the left is 20px, and the font on the right is 12px (default editor font size) according to dev tools.

I've disabled any settings related to font - ligatures, fontFamily, etc. 

![Screen Shot 2020-07-24 at 11 25 06 AM](https://user-images.githubusercontent.com/6858374/88418623-12610b80-cda1-11ea-91a0-5e6b55cf606d.png)
